.travis.yml

```diff
@@ -1,8 +1,6 @@
 sudo: false
 language: node_js
 node_js:
-  - "0.10"
-  - "0.12"
```

Comment:
I disagree with that comment, we can't drop compatible on a patch release.  cc @justinbmeyer